src/views/layouts/main.js

```diff
@@ -11,7 +11,7 @@ const landingHeader = data => `
 
 const mainHeader = data => `
 <header>
-  <a href='/user/breaches'><img class='monitor-logo' src='images/monitor-logo-transparent.png' width='220' height='40'></a>
+  <a href='/user/breaches'><img class='monitor-logo' src='/images/monitor-logo-transparent.png' width='220' height='40'></a>
```

Comment:
Ah – I made all these path changes in [my branch too](https://github.com/mozilla/blurts-server/pull/2688/files#diff-e0482d58197a5dd0faa98700a3ee1b5368f922040ac9e494029282089a3e91ca)
I also added additional code on the same line, so it's likely this will result in conflict.  

Might be best to remove these from your PR and keep in mine?

Reply:
Done